index.js

```diff
@@ -21,19 +30,27 @@ app.get('/game', (req, res) => {
  |         /     \\
  |
 `
-    })
+    };
+    
+    MapDataGames.set(initGame.id, initGame);
+    WriteObjectToJson({path:'./assets/saved-games.json'}, ConvertJson.strMapToObject(MapDataGames));
+    res.send(initGame);    
 })
 
 app.get('/game', (req, res) => {
-    Game.create()
+    
+    var dato =Game.create()

```

Comment:
1. what "dato" means?
2. what is the purpose of this variable? Game.create return a promise, why should we store this?